source/docs/software/support/support-resources.rst

```diff
@@ -0,0 +1,41 @@
+Support Resources
+=================
+
+In addition to the documentation here, there are a variety of other resources available to FRC teams to help understand the Control System and software.
+
+Other Documentation
+-------------------
+
+In addition to this site there are a few other places teams may check for documentation:
+
+-  `NI FRC Community Documents Section <https://decibel.ni.com/content/community/academic/student_competitions/frc?view=documents>`__
+-  `USFIRST.org Technical Resources Page <http://www.usfirst.org/roboticsprograms/frc/2013-Technical-Resources>`__
```

Comment:
https://www.firstinspires.org/resource-library?flagged=All&combine=&field_content_type_value%5B0%5D=first_robotics_competition&field_resource_library_tags_tid=171&sort_by=created_1

Is the correct link

Reply:
This still needs to be fixed.